webpack/assets/javascripts/react_app/common/Tour/BasicWrappedComponent.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { noop } from '../helpers';
+
+class BasicWrappedComponent extends Component {
+  componentDidMount() {
+    this.props.runTour();
```

Comment:
Is there a way to simplify this so we don't need this file?  Seems like a waste if it's just to call `runTour()` in  `componentDidMount`